target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -200,7 +200,8 @@ ramips_setup_macs()
 	unielec,u7628-01-16m|\
 	wavlink,wl-wn575a3|\
 	wiznet,wizfi630s)
-		wan_mac=$(macaddr_add "$(mtd_get_mac_binary factory 0x28)" 1)
+		wan_mac=$(mtd_get_mac_binary factory 0x4)
```

Comment:
As stated in the other comment, please report what's the assignment on OEM firmware; maybe compile a table like here:
https://openwrt.org/docs/guide-developer/device-support-policies#mac_addresses
This can be added to the commit message then.

Apart from that, lan_mac is redundant here, as this is already done by the ethernet node in DTS.

BTW, apart from those points, note that you change the adresses for four independent devices here!